src/components/NewsGroup.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DateNewsCard, IconHeading, Button } from '@baltimorecounty/dotgov-components';
+const NewsGroup = (props) => {
+	const { Newsitems = [] } = props;
+	return (
+		<React.Fragment>
+            <div className="row">
+				{Newsitems.map((Newsitem) => (
+                    <div className="col">
+					<DateNewsCard
+						key={Newsitem.id}
+						date={Newsitem.date}
+						headline={Newsitem.headline}
+						snippet={Newsitem.snippet}
+						link={Newsitem.link}
+					/>
+                    </div>
+				))}
+			</div>
+			 <div  style={{ display: 'flex', justifyContent: 'center', padding: '40px' }}>
+				<Button text="VIEW ALL NEWS" className="dg_button-secondary" />
```

Comment:
the text in this case just needs to be "View all news" the css will handle the casing.

Reply:
done